src/editors/xcode.ts

```diff
@@ -0,0 +1,87 @@
+import os from "os";
+import fs from "async-file";
+import path from "path";
+import { CommandExists } from "../lib/command-exists";
+import "./editor";
+
+export default class Xcode implements Editor {
+  private commandExists = new CommandExists();
+
+  public get name(): string {
+    return "xed";
```

Comment:
Typo, should be `Xcode`. Also, remove `displayName`.

Reply:
`name` could be used for the `key`